features/preferences/impl/src/main/res/values/localazy.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+  <string name="screen_advanced_settings_developer_mode">"Developer mode"</string>
+  <string name="screen_advanced_settings_developer_mode_description">"By enabling you get access to features and functions for developers."</string>
```

Comment:
Either add a comma after "by enabling", or I think a better string would be "Enable to have access to features and functionality for developers"